### ISSUE TITLE
Failed to resolve multi module Android library

### ISSUE THREAD

**Issue by darsh2** [jitpack/jitpack.io#451]
I am unable to add library module of this repository - https://github.com/darsh2/MultipleImageSelect as a dependency in my project.

I tried using compile 'com.github.darsh2.MultipleImageSelect:multipleimageselect:v0.0.3' and
with 'com.github.darsh2.MultipleImageSelect:multipleimageselect:64f06aa' (latest commit), both to no avail.

**Comment by jitpack-io** [comment]
Hi

What about `compile 'com.github.darsh2:MultipleImageSelect:64f06aa'` ?

**Comment by darsh2** [comment]
By default, is the sample module is ignored in the gradle build by jitpack? What about the screenshots folder? That should be excluded - I'm assuming that it would be excluded, right?

**Comment by jitpack-io** [comment]
Yes, by default only library projects are built by JitPack - those that have `apply plugin: 'com.android.library'` in their gradle file.

It's also possible to build non-library projects by including this plugin https://github.com/dcendents/android-maven-gradle-plugin#note-on-releases
